android/src/main/java/com/thebluealliance/androidclient/activities/HomeActivity.java

```diff
@@ -55,7 +55,8 @@
 
     private static final String MAIN_FRAGMENT_TAG = "mainFragment";
 
-    @Inject TBAStatusController mStatusController;
+    @Inject
+    TBAStatusController mStatusController;
```

Comment:
Code -> Reformat Code doesn't seem to be fixing this, isn't that the whole point of defining a code style?